mattermost-plugin/webapp/src/components/boardSelector.tsx

```diff
@@ -116,12 +116,12 @@ const BoardSelector = () => {
     if (showLinkBoardConfirmation?.channelId !== '') {
         confirmationSubText = intl.formatMessage({
             id: 'boardSelector.confirm-link-board-subtext-with-other-channel',
-            defaultMessage: 'Linking the "{boardName}" board to this channel would give all members of this channel "Editor" access to the board.\n\nAdditionally, this board is linked to another channel, and will be unlinked from the other channel when you link it here.\n\nAre you sure you want to link it?'
-        }, {boardName: showLinkBoardConfirmation?.title})
+            defaultMessage: 'Linking the "{boardName}" board to this channel would give all members of this channel "Editor" access to the board.{lineBreak} Additionally, this board is linked to another channel, and will be unlinked from the other channel when you link it here.'
```

Comment:
```suggestion
            defaultMessage: 'When you link "{boardName}" to the channel, all members of the channel (existing and new) will be able to edit it.{lineBreak} This board is currently linked to another channel. It will be unlinked if you choose to link it here.'
```